src/connection.ts

```diff
@@ -141,6 +141,33 @@ export class Connection {
         return data.payload;
     }
 
+    async prepareUsdcWithdraw(authToken: string, params: WithdrawUsdcRequest): Promise<PreparedWithdrawPayload> {
```

Comment:
let vendor these usdc stuff into some `src/plugin/usdc_bridger.ts` imho

we do not want them to populate our root connection class.